h/templates/layouts/account.html.jinja2

```diff
@@ -15,14 +15,27 @@
 {% block content %}
   <div class="content paper">
     {% include "h:templates/includes/logo-header.html.jinja2" %}
-    <ul class="nav nav-tabs">
-      {% for route, title in nav_pages %}
-        <li{% if route == page_route %} class="active"{% endif %}>
-            <a href="{{ request.route_url(route) }}">{{ title }}</a>
-        </li>
-      {% endfor %}
-    </ul>
-
+    {% if feature('activity_pages') %}
+      <div class="form content">
+        <h2 class="form-header">
+          <ul class="form-nav">
```

Comment:
No. It might sense to make this a `<nav>` element, since this is a list of navigational links.